examples/containerlab/README.md

```diff
@@ -0,0 +1,60 @@
+# Containerlab example
+
+This folder contains a simple containerlab topology which is using the image created in the main README.
+The purpose of this lab is to test the creation of bond interface with the API.
+
+## Deployement
+
+Deploy the topology:
+
+```
+sudo clab deploy -t topology.clab.yml
+```
+
+## Manual setup
+
+The `setup_hosts` script configure the bonding automatically, and assign an ip address for the `bond0` interface on each side.
+
+```
+bash setup_hosts
+```
+
+## Setup through API
+
+The following instructions should be done for both container `R1` and `R2`.
+
+First go to the following url to access the swagger api:
+
+```
+http://CONTAINER_IP:8080/api/v2/docs
+```
+
+Then create a new interface for the POST method `/interfaces` with the following parameters:
```

Comment:
This is nice, I would go one step further and actually show the curl commands, so that it can be copy pasted directly.  (You can get the curl commands from the swagger as well btw)